dotlottie-rs/src/dotlottie_player.rs

```diff
@@ -238,6 +255,16 @@ impl DotLottieRuntime {
         self.dotlottie_manager.manifest()
     }
 
+    pub fn size(&self) -> (u32, u32) {
+        (self.renderer.width, self.renderer.height)
+    }
```

Comment:
can you remind me why did we add this function ? is it related to this https://github.com/LottieFiles/dotlottie-rs/pull/180 ?

Reply:
Its not releated to that issue. The problem was If I load a new animation in a `State` (animationId is there), I need the size to be able to call load_animation